split cli into modules

mikz opened the pull request:
so each command is its own file

mikz committed:
splig cli into modules

hallelujah reviewed (approved):
But you also need to modify the rockspec

mikz commented:
Good point.

mikz committed:
fix rockspec

- it points to scm
- can be built both with `luarocks build` and `luarocks make`
